packages/SwingSet/misc-tools/replay-transcript.js

```diff
@@ -359,7 +359,7 @@ async function replay(transcriptFile) {
     }
     syscallResults = {};
     if (divergent && !argv.ignoreConcurrentWorkerDivergences) {
-      throw new Error('Divergent execution between workers');
+      throw Error('Divergent execution between workers');
```

Comment:
Why do this in files that aren't marked @jessy-check? What keeps them from coming back?

Reply:
For general stylistic consistency, and as a step towards being able to add @jessie-check to more files. Even for the files we aren't yet able to mark @jessie-check, we want as much stylistic consistency as practical with those we do.

Reply:
I've been doing `Error(...)` instead of `new Error(...)` for as long as I can remember.